### PR TITLE
Matchmaker violation wording change

### PR DESCRIPTION
**Pull request by maudlin27**
Makes the matchmaker wording slightly less alarming

### CODE REVIEW

**Comment by Askaholic** [comment]
Looks good to me. There's another test that needs to be updated though.

**Comment by maudlin27** [comment]
Updated for the other test so it has consistent wording (although getting different errors with the unit-test check now)

File: tests/unit_tests/test_ladder_service.py [hunk 1]
```diff
@@ -217,8 +217,9 @@ async def test_start_game_timeout(
         mock.call({
             "command": "notice",
             "style": "info",
-            "text": "You have received 1 violations for failing to connect "
-            "to matchmaker games."
+            "text": "You have caused a matchmaking connection failure 1 time(s). "
+                    "Multiple failures result in temporary time-outs from matchmaker. "
+                    "Please seek support on the forums or discord for persistent issues."
```
Comment (Askaholic) [inline tests/unit_tests/test_ladder_service.py:222]: Can you format the multiline strings like this.
```suggestion
            "text": (
                "You have caused a matchmaking connection failure 1 time(s). "
                "Multiple failures result in temporary time-outs from matchmaker. "
                "Please seek support on the forums or discord for persistent issues."
            )
```

File: tests/unit_tests/test_ladder_service.py [hunk 2]
```diff
@@ -273,8 +274,9 @@ async def wait_forever(*args, **kwargs):
...
```
Comment (Askaholic) [inline tests/unit_tests/test_ladder_service.py:279]: Here as well

**Review by Askaholic — COMMENTED** [review]
I think the test fail is probably a fluke. Those integration tests tend to be a little flaky.

File: tests/unit_tests/test_ladder_service.py (hunk 2 above)
Comment (maudlin27) [reply tests/unit_tests/test_ladder_service.py]: Added the brackets/format

Comment (Askaholic) [reply tests/unit_tests/test_ladder_service.py]: Aaah, sorry you didn't format it quite like I said. There should only be 1 additional level of indentation after the parens.

Comment (maudlin27) [reply tests/unit_tests/test_ladder_service.py]: Sorry hadn't realised the importance of indents in python (assumed it was just a code readability thing) - Is the updated commit what you had in mind?

Comment (Askaholic) [reply tests/unit_tests/test_ladder_service.py]: Yep that looks good! In this case it is just a code readability thing. We use one level of indentation everywhere in the code base so I'm just trying to keep it consistent.

**Pull request merged**